content/publishing-yaml/distribution.md

```diff
@@ -197,40 +197,52 @@ echo $ANDROID_FIREBASE_SECRET | base64 --decode > $FCI_BUILD_DIR/android/app/goo
 echo $IOS_FIREBASE_SECRET | base64 --decode > $FCI_BUILD_DIR/ios/Runner/GoogleService-Info.plist
 ```
 
-### Publishing an app using Firebase CLI
+### Publishing an app to Firebase App Distribution
 
-Make sure to encrypt `FIREBASE_TOKEN` as an environment variable. Check [documentation](https://firebase.google.com/docs/cli#cli-ci-systems) for details.
+Make sure to encrypt your Firebase token, it is possible to directly add it under publishing in the encrypted form, or encrypt `FIREBASE_TOKEN` as an environment variable and reference it under publishing. Check [documentation](https://firebase.google.com/docs/cli#cli-ci-systems) for details.
```

Comment:
I'd start with an intro and add links to https://docs.codemagic.io/building/encrypting/ and the instructions to generating/obtaining the token itself, e.g.: To authenticate with Firebase, you need to create a Firebase token, encrypt it and save it to your Codemagic configuration. See how to obtain the token.